ovos_plugin_manager/templates/tts.py

```diff
@@ -47,14 +46,17 @@
 EMPTY_PLAYBACK_QUEUE_TUPLE = (None, None, None, None, None)
 
 
-# TODO move to ovos_utils
-def get_ram_directory(folder):
-    tempfile = MemoryTempfile(fallback=True)
-    path = os.path.join(tempfile.gettempdir(), folder)
-    if not os.path.exists(path):
-        os.makedirs(path)
-    return path
-
+def get_temp_directory(folder):
```

Comment:
final nitpick and its ready to merge, can this be called `get_cache_directory` instead ?

Reply:
Of course, I will rename it, rebase, squash and force push.